py_mcai_worker_sdk/Dockerfile-3.7

```diff
@@ -0,0 +1,21 @@
+FROM rust:1.43-buster as builder
```

Comment:
I suggest to:
- din't change the base image name (keep Dockerfile)
- rename this file with: `Dockerfile.buster`

I think it make more sense to deliver 1 image with multiplie python version using pipenv.

Reply:
Makes sense !